cmd/thanos/rule_test.go

```diff
@@ -48,3 +48,27 @@ func Test_parseFlagLabels(t *testing.T) {
 		testutil.Equals(t, err != nil, td.expectErr)
 	}
 }
+
+func Test_validateTemplate(t *testing.T) {
```

Comment:
I think it would be better to test the actual `TableLinkForExpression` method instead (or in addition), what do you think?

Reply:
Hi @matej-g , I have updated the test.